sdk/python/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -ex
+
+# Copyright 2020 kubeflow.org
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# The scripts creates the KFP-to-Tekton compiler package.
+#
+# Usage:
+#   ./build.sh [output_file]
+
+
+target_archive_file=${1:-kfptekton.tar.gz}
```

Comment:
```suggestion
target_archive_file=${1:-kfptekton-sequential.tar.gz}
```

Reply:
@animeshsingh -- this file is the Python install package (`pip install kfptekton.tar.gz`) for the KFP-Tekton SDK/compiler. This is not the (default) filename for a sequential pipeline. 

If you want to the installable filename to reflect the current limitation of only supporting sequential pipeline compilation, then we would have to change it again after my next change to enable parallel pipelines.